log4j-core/src/main/java/org/apache/logging/log4j/core/impl/ThrowableProxy.java

```diff
@@ -104,8 +104,8 @@ public ThrowableProxy(final Throwable throwable) {
         this.name = throwable.getClass().getName();
         this.message = throwable.getMessage();
         this.localizedMessage = throwable.getLocalizedMessage();
-        final Map<String, ThrowableProxyHelper.CacheEntry> map = new HashMap<>();
         final Stack<Class<?>> stack = StackLocatorUtil.getCurrentStackTrace();
```

Comment:
Note (unrelated to this commit): This doesn't quite work when we use the AsyncLoggerContextSelector because ThrowableProxy instances are created lazily on the background thread, where the current stack isn't helpful.